SPECS/krb5/krb5.spec

```diff
@@ -12,6 +12,11 @@ Requires:       openssl
 Requires:       e2fsprogs-libs
 BuildRequires:  openssl-devel
 BuildRequires:  e2fsprogs-devel
+
+%if %{with_check}
+BuildRequires:  iana-etc
```

Comment:
Will trigger toolchain rebuilds on `dev` since `iana-etc` is not in the toolchain.

This should either be reverted or `iana-etc` should be added to the toolchain.

Reply:
Will remove from this PR.